tcms/testruns/static/testruns/js/get.js

```diff
@@ -239,6 +235,28 @@ function renderTestCaseInformation(testExecutions, testExecutionCaseIds) {
         })
     })
 
+    testExecutions.forEach(testExecution => {
```

Comment:
Similar loop on line 190 (ends on 235). IMO the jsonRPC call for history can be inserted after line 234.